src/lib/transform.js

```diff
@@ -62,14 +62,31 @@ function transformRelatedModel(field, prop) {
   return transformer;
 }
 
-function transformGrantModel(field) {
+function transformGrantModel(prop, secondaryProp) {
```

Comment:
Maybe it doesn't have to be this exact PR, but I can't help thinking there is more abstracted way to write this that could reuse ```transformRelatedModel``` - like a simpler function that just passes the activity recipients down into it (so instance[field] gets the grants, etc).

Reply:
Spoke with Josh we will break this into two functions so its easier to read and understand later.